gui/RemoveItem.java

```diff
@@ -238,34 +230,25 @@ public void setup() {
 		rlist.add(new StringEntry(Minecraft.getMinecraft().thePlayer.getDisplayNameString(), (StringEntry entry,
 				DisplayList dlist, int mouseX, int mouseY) -> entryClicked(entry, dlist, mouseX, mouseY)));
 
-		rosterDisplayList = new ScrollableDisplayList((int) (width * .15), (int) (height * .275), width / 3, 100, 15,
+		rosterDisplayList = new ScrollableDisplayList((int) (width * .15), (int) (height * .35), width / 3, 100, 15,
 				rlist);
 		rosterDisplayList.setId("roster");
 		registerComponent(rosterDisplayList);
 
-		userBox = new TextBox((int) (width * .15), (int) (height * .725), width / 4, 20, "User").setId("user")
+		userBox = new TextBox((int) (width * .15), (int) (height * .8), width / 4, 20, "User").setId("user")
 				.setTextChangedListener((TextBox textbox, String previousText) -> textChanged(textbox, previousText));
 		registerComponent(userBox);
 
-		amountBox = new TextBox((int) (width * .45) - 16, (int) (height * .725), 30, 20, "Amt").setId("amt")
+		amountBox = new TextBox((int) (width * .45) - 16, (int) (height * .8), 30, 20, "Amt").setId("amt")
 				.setTextChangedListener((TextBox textbox, String previousText) -> textChanged(textbox, previousText));
 		registerComponent(amountBox);
 
-		itemBox = new TextBox((int) (width * .5), (int) (height * .725), width / 4, 20, "Item").setId("item")
+		itemBox = new TextBox((int) (width * .5), (int) (height * .8), width / 4, 20, "Item").setId("item")
 				.setTextChangedListener((TextBox textbox, String previousText) -> textChanged(textbox, previousText));
 		registerComponent(itemBox);
 
-		registerComponent(new Button((int) (width * .7875) - 10, (int) (height * .725), 40, 20, "Remove")
-				.setClickListener(but -> removeItemFromPlayer()));
-
-		registerComponent(new Button((int) (width * .175) - 10, (int) (height * .825), 100, 20, "Clear Roster Inv")
-				.setClickListener(but -> clearAllPlayerInventorys()));
-
-		registerComponent(new Button((int) (width * .4225) - 10, (int) (height * .825), 90, 20, "Clear Player Inv")
-				.setClickListener(but -> clearPlayerInventory()));
-
-		registerComponent(new Button((int) (width * .645) - 10, (int) (height * .825), 102, 20, "Remove Roster Item")
-				.setClickListener(but -> removeItemFromPlayer()));
```

Comment:
No a lot of the changes you are seeing are not things I did. I just copied the server mod a couple weeks ago and this is what everything was like.